tests/integration/targets/aci_l3out_bgp_peer/tasks/main.yml

```diff
@@ -225,37 +225,37 @@
 - name: verify BGP peer has been created with correct attributes (version >= 4)
   assert:
     that:
-    - add_eth_bgp_peer.current.0.bgpPeerP.attributes.dn == "uni/tn-ansible_tenant/out-ansible_l3out/lnodep-ansible_node_profile/lifp-ansible_interface_profile/rspathL3OutAtt-[topology/pod-1/paths-201/pathep-[eth1/15]]/peerP-[192.168.50.2]"
-    - add_eth_bgp_peer.current.0.bgpPeerP.attributes.addr == "192.168.50.2"
-    - add_eth_bgp_peer.current.0.bgpPeerP.attributes.addrTCtrl == "af-ucast"
-    - add_eth_bgp_peer.current.0.bgpPeerP.attributes.adminSt == "enabled"
-    - add_eth_bgp_peer.current.0.bgpPeerP.attributes.ctrl == "nh-self,send-com,send-ext-com"
-    - add_eth_bgp_peer.current.0.bgpPeerP.attributes.peerCtrl == "bfd"
-    - add_eth_bgp_peer.current.0.bgpPeerP.attributes.ttl == "2"
-    - add_eth_bgp_peer.current.0.bgpPeerP.attributes.annotation == 'orchestrator:ansible'
+      - add_eth_bgp_peer.current.0.bgpPeerP.attributes.dn == "uni/tn-ansible_tenant/out-ansible_l3out/lnodep-ansible_node_profile/lifp-ansible_interface_profile/rspathL3OutAtt-[topology/pod-1/paths-201/pathep-[eth1/15]]/peerP-[192.168.50.2]"
+      - add_eth_bgp_peer.current.0.bgpPeerP.attributes.addr == "192.168.50.2"
+      - add_eth_bgp_peer.current.0.bgpPeerP.attributes.addrTCtrl == "af-ucast"
+      - add_eth_bgp_peer.current.0.bgpPeerP.attributes.adminSt == "enabled"
+      - add_eth_bgp_peer.current.0.bgpPeerP.attributes.ctrl == "nh-self,send-com,send-ext-com"
+      - add_eth_bgp_peer.current.0.bgpPeerP.attributes.peerCtrl == "bfd"
+      - add_eth_bgp_peer.current.0.bgpPeerP.attributes.ttl == "2"
+      - add_eth_bgp_peer.current.0.bgpPeerP.attributes.annotation == 'orchestrator:ansible'
```

Comment:
The additional space is not necessary, why add it everywhere?
The - can be directly under the : in YAML